### PR TITLE
cleanup CODEOWNERS

### PR DESCRIPTION
**Pull request by mihaipopescu**
I honestly don't have any spare cycles to contribute to this repository.
Keep up the good work!

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,11 +1 @@
...
-
-.editorconfig   @mihaipopescu
-
-# CI
-.travis.yml     @mihaipopescu
-appveyor.yml    @mihaipopescu
-
-# emscripten
-#/emscripten/    @CDanU
-#/src/uncrustify_emscripten.cpp @CDanU
```
Comment (micheleCTDEAdmin) [inline CODEOWNERS:11]: Can you amend to avoid removing other people's name? I guess it was an oversight.

**Review by micheleCTDEAdmin — CHANGES_REQUESTED** [review]
Please amend as per comment.

File: CODEOWNERS (hunk 1 above)
Comment (mihaipopescu) [reply CODEOWNERS:11]: I can separate into a different commit but those lines are commented out. We have the git history for that.

Comment (micheleCTDEAdmin) [reply CODEOWNERS:11]: Fair enough :-)

**Review by micheleCTDEAdmin — APPROVED** [review]

**Comment by gmaurel** [comment]
as "admin" I wanted to say "Thanks to you for the many PR"

**Pull request merged**